### DISCUSSION TITLE
Missing ARM version of rabbitmq:3.12-management-alpine

### DISCUSSION THREAD

**Discussion by jcolfej** [docker-library/rabbitmq#654] [Q&A]
Hello !

I'm opening an issue because it seems that the `rabbitmq:3.12-management-alpine` image doesn't have an ARM-compatible version.
```no matching manifest for linux/arm64/v8 in the manifest list entries```

Is this a temporary problem, or is it intentional for the moment?

**Comment by realsuayip** [comment]
`rabbitmq:3.12-alpine` is missing too.

**Comment by lukebakken** [comment]
Both are listed here:

https://hub.docker.com/r/arm64v8/rabbitmq/

Please provide the _full_ command you are running and its output that demonstrates the issue you're seeing.

**Reply by yosifkit** [reply]
It seems like a few arches were failing to build after https://github.com/docker-library/rabbitmq/pull/641 since they don't create a `lib64` (so the `COPY` fails because it doesn't exist. New fix in https://github.com/docker-library/rabbitmq/pull/655.

**Reply by lukebakken** [reply]
Thank you. I forgot where to check the logs for the builds, would you remind me where they are?

**Reply by tianon** [reply]
There are badges in the `README` of this repo (which might be easier for you to find/remember next time), but we generally go to https://doi-janky.infosiftr.net/job/multiarch/view/images/view/rabbitmq/ :smile:

That might change in the future, but for now that's the place. :bow:

**Reply by lukebakken** [reply]
Thank you!
